boards/native/Makefile.dep

```diff
@@ -1,9 +1,13 @@
 ifneq (,$(filter defaulttransceiver,$(USEMODULE)))
-    ifeq (,$(filter ng_netif,$(USEMODULE)))
-        USEMODULE += nativenet
+    USEMODULE += nativenet
+    ifeq (,$(filter netdev_base,$(USEMODULE)))
```

Comment:
you should know this... I just copied it from the Makefile.deps of the iot-lab and the samr21...